.claude/commands/shared/reference-patterns.yml

```
@@ -0,0 +1,105 @@
+  Self_Contained: "Each command has complete information"
+```
+
+## Performance Optimizations
+
+```yaml
+Reference_Performance:
+  Caching:
+    Strategy: "Cache resolved references for session duration"
+    Invalidation: "Clear cache when shared files change"
+    Memory_Usage: "Limit cache size to prevent memory issues"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)